kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
```

Comment:
not really bothered, but just a fyi, I purposefully used `md` rather than `markdown` because this matches the abbreviation used by the markdown library (see `md.parser`, `md.processor` etc in `KordacExtension.py`)